darglint/docstring/base.py

```diff
@@ -10,39 +9,8 @@
     Iterable,
 )
 
-
-class DocstringStyle(enum.Enum):
-    GOOGLE = 0
-    SPHINX = 1
-    NUMPY = 2
-
-    @classmethod
-    def from_string(cls, style):
-        style = style.lower().strip()
-        if style == 'google':
-            return cls.GOOGLE
-        if style == 'sphinx':
-            return cls.SPHINX
-        if style == 'numpy':
-            return cls.NUMPY
-
-        raise Exception(
-                'Unrecognized style "{}".  Should be one of {}'.format(
-                    style,
-                    [x.name for x in DocstringStyle]
-                )
-            )
-
-
-class Sections(enum.Enum):
-    SHORT_DESCRIPTION = 0
-    LONG_DESCRIPTION = 1
-    ARGUMENTS_SECTION = 2
-    RAISES_SECTION = 4
-    YIELDS_SECTION = 6
-    RETURNS_SECTION = 8
-    VARIABLES_SECTION = 10
-    NOQAS = 13
+from .sections import Sections  # noqa: F401
+from ..strictness import Strictness  # noqa: F401
```

Comment:
what do you say about using [`isort`](https://github.com/pycqa/isort)?
We can create some config that will be the best to use in `darglint` and check imports in travis on every push.

Reply:
@terrencepreilly could you share your opinion about `isort`?

Reply:
I wouldn't be opposed to using `isort`, though I'd rather just add it as a git hook rather than putting it on travis.

There are a lot of housekeeping sort of things I'd like to do for darglint, but just haven't had time.  I'd like to auto-format everything in darglint.  And add some hooks to check things like version numbers, and tags, etc. have been updated.

If you want to, you can add a pre-commit configuration (and add it to the dev dependencies in `pyproject.toml`), and add isort as a hook.  Otherwise, I'll probably get around to it sometime after I finish working on optimizing the parser and finishing numpy integrations.